internal/uvm/vpmem.go

```diff
@@ -103,14 +146,25 @@ func (uvm *UtilityVM) addVPMemDefault(ctx context.Context, hostPath string) (_ s
 		},
 		ResourcePath: fmt.Sprintf(resourcepaths.VPMemControllerResourceFormat, deviceNumber),
 	}
+
 	uvmPath := fmt.Sprintf(lcowDefaultVPMemLayerFmt, deviceNumber)
+	guestSettings := guestrequest.LCOWMappedVPMemDevice{
+		DeviceNumber: deviceNumber,
+		MountPath:    uvmPath,
+	}
+	if v, iErr := readVeritySuperBlock(ctx, hostPath); iErr != nil {
+		log.G(ctx).WithError(err).WithField("hostPath", hostPath).Debug("unable to read dm-verity information from VHD")
```

Comment:
this doesn't error out the call though right, so we'd still make a guest request? I thought the only time we wanted to silently continue was when we get a `ErrSuperBlockReadFailure` error

Reply:
skipping the error here ensures we always have a fallback to the original behavior. whether it's the way to go about is debatable of course.

Reply:
so the idea is that if we fail to read the verity block then we just mount it as a normal vpmem device? But verity wouldn't be setup properly?

Reply:
yes, that's correct. we could also introduce something that will tell us to always enforce it, for example or vice-versa (i.e. ignore failure)

Reply:
I see, so we always try to use dm-verity if possible. I think that might be fine.

Reply:
yup